### ISSUE TITLE
Request: Deadman switch

### ISSUE THREAD

**Issue by jcbit** [bitfinexcom/bitfinex-api-node#509]
#### Issue type
- [ ] bug
- [X ] missing functionality
- [ ] performance
- [ ] feature request

#### Brief description
Would be nice to be able to use the dead man switch. Could you implement it so that you can set that up via an environment variable please?

#### Steps to reproduce
-

##### Additional Notes:
-

**Comment by JacobPlaster** [comment]
Hey @jcbit thanks for opening the issue. You can actually achieve this with the functionality below:

```
const { DMS } = process.env
const ws = bfx.ws(2)

ws.on('open', () => { // wait for socket open
  ws.auth(0, DMS) // & authenticate
})
``` 

Use env variable in order to enable the DMS flag.
```
export DMS=4
```

**Comment by jcbit** [comment]
Thanks for that information! 

I am trying it and wanted to ask what is the code (0) used for? Also, I wanted to use DMS so that if the application crashed all the orders were cancelled automatically. But just now that I tried it I see that my order is still there even though the program ended abruptly. Am I doing something wrong? Already checked and DMS is 4 when calling auth.

**Comment by JacobPlaster** [comment]
The `0` is the `calc` parameter https://docs.bitfinex.com/reference#ws-auth-input-calc.

> But just now that I tried it I see that my order is still there even though the program ended abruptly
Possibly not, basically its really hard to tell when a websocket is closed abruptly since no closing header is sent to the server. The way we get around this is be re-checking all of our open websocket connections every 2-10 mins to see if they are still connected, this means that it could be 2-10 mins before your dead-man-switch is flicked. However, if we receive a 'close' header from the websocket, then the dead-man-switch is flicked instantly.

TL;DR - try waiting a few mins

Let me know if that doesnt work
